tasks/integration/windows.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Set Agent binary path
+  set_fact:
+    datadog_agent_binary_path: "{{ datadog_agent_binary_path_windows }}"
+  when: ansible_facts.os_family == "Windows"
+
+- name: Set Agent user for integration commmand
+  set_fact:
+    integration_command_user: "{{ integration_command_user_windows }}"
+  when: ansible_facts.os_family == "Windows"
```

Comment:
This (and the other when conditions mentioning "Windows") are unnecessary as already checked to call this file, correct?